packages/twenty-front/src/modules/object-metadata/graphql/queries.ts

```diff
@@ -68,6 +68,7 @@ export const FIND_MANY_OBJECT_METADATA_ITEMS = gql`
                 defaultValue
```

Comment:
I'm wondering if we still need to query toRelationMetadata/fromRelationMetadata above

Reply:
we are still using it to get the object names in toObjectMetadata / fromObjectMetadata. or did you have something else in mind?